### PR TITLE
Support load data from disk file

### PR DESCRIPTION
**Pull request by Linary**

### CODE REVIEW

File: hubble-be/src/main/java/com/baidu/hugegraph/controller/load/LoadTaskController.java [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+//        GraphConnection connection = this.connService.get(connId);
+//        if (connection == null) {
+//            throw new ExternalException("graph-connection.not-exist.id", connId);
+//        }
+//        FileMapping fileMapping = this.fmService.get(fileId);
+//        if (fileMapping == null) {
+//            throw new ExternalException("file-mapping.not-exist.id", fileId);
+//        }
+//        return this.service.start(connection, fileMapping);
+//    }
```
Comment (javeme) [inline hubble-be/src/main/java/com/baidu/hugegraph/controller/load/LoadTaskController.java:136]: unused?

File: hubble-be/src/main/java/com/baidu/hugegraph/handler/LoadTaskMonitor.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+//            }
+//        }
+//        task.setFileReadLines(context.newProgress().totalInputReaded());
+//        task.setDuration(context.summary().totalTime());
+//        if (this.service.update(task) != 1) {
+//            throw new InternalException("entity.update.failed", task);
+//        }
+//        this.service.getTaskContainer().remove(task.getId());
+//    }
+//}
```
Comment (javeme) [inline hubble-be/src/main/java/com/baidu/hugegraph/handler/LoadTaskMonitor.java:74]: ditto

File: hubble-be/src/main/java/com/baidu/hugegraph/options/HubbleOptions.java [hunk 3]
```diff
@@ -158,19 +159,35 @@ public static synchronized HubbleOptions instance() {
...
+                    "csv"
+            );
+
+    public static final ConfigOption<Long> UPLOAD_SINGLE_FILE_SIZE_LIMIT =
             new ConfigOption<>(
-                    "indexlabel.rebuild.timeout",
-                    "The timeout in seconds for waiting to create index label.",
-                    rangeInt(-1, Integer.MAX_VALUE),
-                    30
+                    "upload_single_file.size.limit",
```
Comment (javeme) [inline hubble-be/src/main/java/com/baidu/hugegraph/options/HubbleOptions.java:180]: prefer upload_file.single_file_size_limit

File: hubble-be/src/main/java/com/baidu/hugegraph/options/HubbleOptions.java [hunk 4]
```diff
@@ -158,19 +159,35 @@ public static synchronized HubbleOptions instance() {
...
             );
 
-    public static final ConfigOption<Integer> INDEXLABEL_REMOVE_TIMEOUT =
+    public static final ConfigOption<Long> UPLOAD_TOTAL_FILE_SIZE_LIMIT =
             new ConfigOption<>(
-                    "indexlabel.remove.timeout",
-                    "The timeout in seconds for waiting to remove index label.",
-                    rangeInt(-1, Integer.MAX_VALUE),
-                    30
+                    "upload_total_file.size.limit",
```
Comment (javeme) [inline hubble-be/src/main/java/com/baidu/hugegraph/options/HubbleOptions.java:188]: prefer upload_file.total_file_size_limit

File: hubble-be/src/main/java/com/baidu/hugegraph/service/load/LoadTaskService.java [hunk 5]
```diff
@@ -0,0 +1,484 @@
...
+        source.listFormat().endSymbol(listFormat.getEndSymbol());
+        source.listFormat().elemDelimiter(listFormat.getElemDelimiter());
+        return source;
+    }
+
+    private List<com.baidu.hugegraph.loader.mapping.VertexMapping>
+            buildVertexMappings(GraphConnection connection,
+                                FileMapping fileMapping) {
+        int connId = connection.getId();
+        List<com.baidu.hugegraph.loader.mapping.VertexMapping> vMappings =
```
Comment (javeme) [inline hubble-be/src/main/java/com/baidu/hugegraph/service/load/LoadTaskService.java:376]: why not import it

File: hubble-be/src/main/java/com/baidu/hugegraph/service/load/UploadFileService.java [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+import org.springframework.stereotype.Service;
+
+import lombok.extern.log4j.Log4j2;
+
+@Log4j2
+@Service
+public class UploadFileService {
+
+
```
Comment (javeme) [inline hubble-be/src/main/java/com/baidu/hugegraph/service/load/UploadFileService.java:30]: empty lines

File: hubble-be/src/main/java/com/baidu/hugegraph/util/DataTypeUtil.java [hunk 7]
```diff
@@ -0,0 +1,289 @@
...
+                                                   Collection<?> values,
+                                                   DataType dataType) {
+        for (Object value : values) {
+            if (!checkDataType(key, value, dataType)) {
+                return false;
+            }
+        }
+        return true;
+    }
+}
```
Comment (javeme) [inline hubble-be/src/main/java/com/baidu/hugegraph/util/DataTypeUtil.java:289]: import from loader

File: hubble-be/src/main/java/com/baidu/hugegraph/service/load/LoadTaskService.java (hunk 5 above)
Comment (Linary) [reply hubble-be/src/main/java/com/baidu/hugegraph/service/load/LoadTaskService.java:376]: There is an same class in hubble

**Pull request merged**